rts/Sim/Features/Feature.cpp

```diff
@@ -463,6 +464,7 @@ void CFeature::ForcedMove(const float3& newPos)
 	quadField.RemoveFeature(this);
 
 	const float3 oldPos = pos;
+	prevPos = pos;
```

Comment:
why don't you call DisjointInterp here?

Reply:
That is a good point. I also did not update prevPos in the correct movement function.